manifest_staging/charts/aad-pod-identity/templates/podidentityexceptions.yaml

```
@@ -0,0 +1,16 @@
+{{- if not (empty .Values.podIdentityExceptions) -}}
+{{- range $key, $value := .Values.podIdentityExceptions }}
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzurePodIdentityException
+metadata:
+  name: {{ default $key $value.name }}
+  namespace: {{ default $.Release.Namespace $value.namespace }}
+  labels:
+    {{- include "aad-pod-identity.labels" $ | nindent 8 }}
```

we only need 4 spaces as indentation:

```suggestion
    {{- include "aad-pod-identity.labels" $ | nindent 4 }}
```